### PR TITLE
Delete old auth logic

### PR DESCRIPTION
**Pull request by aderrod94**
Deleted pages, components and code that related to the old auth logic.

### CODE REVIEW

File: website/frontend/src/utils/RPCHandler.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import io from 'socket.io-client';
```
Comment (codev0) [inline website/frontend/src/utils/RPCHandler.ts:1]: We already removed socket.io in frontend, so this file is unnecessary

Comment (aderrod94) [reply website/frontend/src/utils/RPCHandler.ts]: done, thank you)

**Review by codev0 — APPROVED** [review]

File: website/frontend/src/store/actions/actions.ts [hunk 2]
```diff
@@ -18,23 +18,17 @@ import {
 
 import { UserAccount, AccountActionTypes } from 'types/account';
 import { ApiActionTypes } from 'types/api';
-import GitRequestService from 'services/GitRequestService';
-import TelegramRequestService from 'services/TelegramRequestService';
 import ProgramRequestService from 'services/ProgramsRequestService';
 import NotificationsRequestService from 'services/NotificationsRequestService';
 
 import ServerRPCRequestService from 'services/ServerRPCRequestService';
-import { GEAR_MNEMONIC_KEY, GEAR_STORAGE_KEY, RPC_METHODS } from 'consts';
```
Comment (osipov-mit) [inline website/frontend/src/store/actions/actions.ts:27]: GEAR_MNEMONIC_KEY don't used

File: website/frontend/src/store/actions/actions.ts [hunk 3]
```diff
@@ -158,45 +146,6 @@ export const generateKeypairAction = () => (dispatch: any) => {
...
-      window.localStorage.setItem(GEAR_STORAGE_KEY, result.result.access_token);
-      dispatch(fetchTokenSuccessAction(result));
-      if (result.result.access_token) {
-        window.location.reload();
-      }
-    })
-    .catch(() => dispatch(fetchTokenErrorAction()));
-};
-
 export const getUserDataAction = () => (dispatch: any) => {
```
Comment (osipov-mit) [inline website/frontend/src/store/actions/actions.ts:149]: no need

File: website/frontend/src/store/reducers/userReducer.ts [hunk 4]
```diff
@@ -11,15 +10,6 @@ const initialState: UserState = {
...
-    case UserActionTypes.FETCH_TOKEN:
-      return { ...state, loading: true, error: null };
-
-    case UserActionTypes.FETCH_TOKEN_SUCCESS:
-      return { ...state, loading: false, error: null, accessToken: action.payload };
-
-    case UserActionTypes.FETCH_TOKEN_ERROR:
-      return { ...state, loading: false, error: action.payload, accessToken: '' };
-
     case UserActionTypes.FETCH_USER:
```
Comment (osipov-mit) [inline website/frontend/src/store/reducers/userReducer.ts:13]: i think it is also no need

File: website/frontend/src/types/user.ts [hunk 5]
```diff
@@ -34,30 +33,13 @@ export interface UserKeypairRPCModel {
...
 export enum UserActionTypes {
-  FETCH_TOKEN = 'FETCH_TOKEN',
-  FETCH_TOKEN_SUCCESS = 'FETCH_TOKEN_SUCCESS',
-  FETCH_TOKEN_ERROR = 'FETCH_TOKEN_ERROR',
   FETCH_USER = 'FETCH_USER',
   FETCH_USER_SUCCESS = 'FETCH_USER_SUCCESS',
   FETCH_USER_ERROR = 'FETCH_USER_ERROR',
   FETCH_USER_KEYPAIR = 'FETCH_KEYPAIR',
   FETCH_USER_KEYPAIR_SUCCESS = 'FETCH_USER_KEYPAIR_SUCCESS',
   FETCH_USER_KEYPAIR_ERROR = 'FETCH_USER_KEYPAIR_ERROR',
```
Comment (osipov-mit) [inline website/frontend/src/types/user.ts:41]: same

File: website/frontend/src/types/user.ts [hunk 6]
```diff
@@ -88,19 +70,11 @@ interface TransferBalanceAction {
...
 export type UserAction =
-  | FetchTokenAction
-  | FetchTokenSuccessAction
-  | FetchTokenErrorAction
   | FetchUserAction
   | FetchUserErrorAction
   | FetchUserSuccessAction
   | FetchUserKeypairAction
   | FetchUserKeypairSuccessAction
   | FetchUserKeypairErrorAction
```
Comment (osipov-mit) [inline website/frontend/src/types/user.ts:79]: same

**Review by osipov-mit — CHANGES_REQUESTED** [review]

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**